src/civetweb.c

```diff
@@ -1076,6 +1096,7 @@ static struct mg_option config_options[] = {
     {"ssl_verify_depth", CONFIG_TYPE_NUMBER, "9"},
     {"ssl_default_verify_paths", CONFIG_TYPE_BOOLEAN, "yes"},
     {"ssl_forward_secrecy", CONFIG_TYPE_BOOLEAN, "yes"},
+    {"ssl_cipher_list", CONFIG_TYPE_STRING, "ALL"},
```

Comment:
I would prefer to have NULL as a default, and call `SSL_CTX_set_cipher_list` only `if (ctx->config[SSL_CIPHER_LIST] != NULL)`. This would keep the current behavior.

I'm not sure if the openSSL default is ALL, and will always be ALL, or they drop some weak ciphers in the default.